backends/tc/tc.def

```diff
@@ -446,12 +443,12 @@ class TCKey {
         emitValue = true;
     }
     toString {
-        std::string tckeyInstance = " " + keyAttribute + " " + keyName + " ptype bit" + Util::toString(bitwidth);
+        std::string tckeyInstance = absl::StrCat(" ", keyAttribute.c_str(), " ", keyName.c_str(), " ptype ", type.c_str());
```

Comment:
what happened with `bitwidth`? See above about `string_view`

Reply:
Changing bitwidth to ptype was original PR change.

Reply:
Ok. Then why have you resolved it without addressing `string_view` comment?